### ISSUE TITLE
How to block a user?

### ISSUE THREAD

**Issue by htrxuan** [AgoraIO/Signaling#13]
Hi,

How to block a user in channel? (one admin of channel, all others are user), so admin can block any bad users, could I do this with agora?

**Comment by plutoless** [comment]
you can implement your own logic to send a ban msg to any user who you want to block. And the user needs to handle this message and leave the channel. That's how it's typically done with our RTM/Signaling SDK.

**Comment by htrxuan** [comment]
no, I want to set an admin, so this guy can ban any user ? Could I do this?

**Comment by plutoless** [comment]
there's no admin concept in our RTM/Signaling SDK.

**Comment by htrxuan** [comment]
i know, but anyway to ban a user after they joined to a channel?

**Comment by plutoless** [comment]
seems to be the only way. you can also deploy your own service, and send restful request to check if a user is able to join a channel or not

**Comment by plutoless** [comment]
seems to be the only way. you can also deploy your own service, and send restful request to check if a user is able to join a channel or not

**Comment by htrxuan** [comment]
no, it's not good idea. If I need to create, I will use other service for text chat instead of Agora Signaling.
